client/app.jsx

```diff
@@ -51,6 +54,8 @@ export default class App extends React.Component {
       size: 10,
       undoStack: [],
       redoStack: [],
+      monthNames: ['January', 'February', 'March', 'April', 'May', 'June',
+        'July', 'August', 'September', 'October', 'November', 'December'],
```

Comment:
Not important here, but for larger projects I'd suggest using the `Intl.DateTimeFormat()` API: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

Reply:
gotcha thanks